app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-confirm-step.tsx

```diff
@@ -0,0 +1,51 @@
+import { Alert } from '@18f/identity-components';
+import { FormStepsContext, FormStepsContinueButton } from '@18f/identity-form-steps';
+import { t } from '@18f/identity-i18n';
+import type { FormStepComponentProps } from '@18f/identity-form-steps';
+import { Modal } from '@18f/identity-modal';
+import PersonalKeyStep from '../personal-key/personal-key-step';
+import PersonalKeyInput from './personal-key-input';
+import type { VerifyFlowValues } from '../..';
+
+interface PersonalKeyConfirmStepProps extends FormStepComponentProps<VerifyFlowValues> {}
+
+function PersonalKeyConfirmStep(stepProps: PersonalKeyConfirmStepProps) {
+  const { registerField, errors } = stepProps;
+
+  return (
+    <>
+      <FormStepsContext.Provider value={{ isLastStep: false, onPageTransition() {} }}>
+        <PersonalKeyStep {...stepProps} />
+      </FormStepsContext.Provider>
+      <Modal>
```

Comment:
Follow-up: Add "key badge" icon to show at the top of the confirmation modal (thinking it could be a prop of `Modal` once #6198 lands) (LG-6159).